src/lib/validation/index.ts

```diff
@@ -7,7 +7,7 @@ import isOfferValidityDatesValid from "./isOfferValidityDatesValid";
 import isRedemptionDatesValid from "./isRedemptionDatesValid";
 import returnPeriodValue from "./returnPeriodValue";
 
-Yup.addMethod<Yup.ArraySchema<Yup.BaseSchema<Dayjs>>>(
+Yup.addMethod(
```

Comment:
@albertfolch-redeemeum please confirm if the change doesn't break the logic.
Even if I've tested on my own, I'm not 100% sure about the consequence of it

Reply:
also with this yup upgrade I see you can do this in src/components/product/utils/validationSchema.ts
```
          // .min(2, validationMessage.required)
          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
          // @ts-ignore
          .isRedemptionDatesValid()
```
to
```
          // .min(2, validationMessage.required)
          .isRedemptionDatesValid()
```

Reply:
done